chemie/corporate/templates/corporate/interview_detail.html

```diff
@@ -39,31 +39,40 @@ <h1 class="heading">{{ interview.title }}</h1>
       {{ interview.text | safe }}
     </div>
     <div class="col-12 col-lg-4 order-1 order-lg-3">
-      <img class="img-fluid" style="margin-bottom: 30px; width: 100%; max-width: 60vw" src="{{ interview.picture.url }}">
+      <img class="img-fluid" style="margin-bottom: 30px; width: 100%; max-width: 60vw"
+           src="{{ interview.picture.url }}">
     </div>
     <div class="col-12 col-lg-12 order-4">
       {% if perms.corporate.delete_interview %}
-        <a type="button" class="button-warning"
-           onclick="deleteInterviewPopup({{ interview.id }})"
-           id="deleteInterview{{ interview.id }}"
-           data-url="{% url "corporate:interview_delete" interview.id %}">
-           Slett intervju
-         </a>
+      <a type="button" class="button-warning"
```

Comment:
Place the edit and delete button side by side for a cleaner look and consider moving them to the right side of the page. 
Other than that the functionality works well.